### PR TITLE
Front Facing text changes (Branding)

### PR DESCRIPTION
**Pull request by nZambi-**
Just changing some of the Front Facing text. (also sneaked  in a zip removal... sorry) This just changes the Welcome text and removes/replaces the Emby Beta text (which prompted the user to visit emby.media )

### CODE REVIEW

File: source/ServerScreens.brs [hunk 1]
```diff
@@ -9,11 +9,11 @@
...
 
-    header = "Welcome to Emby"
+    header = "Welcome to Jellyfin"
     paragraphs = []
-    paragraphs.Push("With Emby you can easily stream videos, music and photos to Roku and other devices from your Emby Server.")
-    paragraphs.Push("To begin, please make sure your Emby Server is currently running. For information on how to download and install Emby Server, visit:")
-    paragraphs.Push("http://www.emby.media")
+    paragraphs.Push("With Jellyfin you can easily stream videos, music and photos to Roku and other devices from your Jellyfin Server.")
+    paragraphs.Push("To begin, please make sure your Server is currently running. For information on how to download and install Emby Server, visit:")
+    paragraphs.Push("http://www.jellyfin.media")
```
Comment (andrewrabert) [inline source/ServerScreens.brs]: Remove the prefixing `www.`.

File: source/ViewController.brs [hunk 2]
```diff
@@ -1851,16 +1831,8 @@ Sub CheckDisplayBetaHint()
...
-    if message = "" or message.left(1) = "0" then 
-        Print "No message to display"
-        return
-    end if
-
-    message = message.split("|")
-    title = FirstOf(message[0], "Try our fresh new app!")
-    msg = FirstOf(message[1], "Preview the upcoming much improved Emby Roku app now! �A beautifully redesigned interface that allows you to view your Emby content on your Roku like never before. �Try it now by installing the private preview channel at http://emby.media/roku")
+	title = "This is an Early Beta"
+	msg = "Check for the latest information on this, and everything Jellyfin, at https://github.com/jellyfin"
```
Comment (andrewrabert) [inline source/ViewController.brs]: Change url to `https://jellyfin.media`.

**Review by andrewrabert — CHANGES_REQUESTED** [review]

**Comment by andrewrabert** [comment]
FYI - I just changed `https://jellyfin.media` from redirecting to the server repo to now redirect to the Jellyfin GitHub org. Once our branding is finalized, it will redirect to a website.

**Comment by nZambi-** [comment]
I think that has it for this one.  I'll make sure everything I find from here on points to https://jellyfin.media

**Review by andrewrabert — APPROVED** [review]

**Pull request merged**